BUILD

```diff
@@ -18,13 +19,31 @@ buildifier(
     mode = "fix",
 )
 
+# Usage:
+#  - bazel run //:requirements.update
 compile_pip_requirements(
     name = "requirements",
     src = "requirements.txt",
     requirements_txt = "requirements_lock.txt",
 )
 
+# Usage:
+#  - bazel run //:venv venv
 py_venv(
     name = "venv",
     deps = all_requirements,
 )
+
+# Usage:
+#  - Build all targets with flags
+#    - bazel build --config pico //emb/...
+#  - bazel run //:refresh_compile_commands
+refresh_compile_commands(
+    name = "refresh_compile_commands",
+
+    # Specify the targets of interest.
+    # For example, specify a dict of targets and any flags required to build.
+    targets = {
+        "//emb/...": "",
```

Comment:
Why add this if it's empty? I haven't used it before so just asking

Reply:
it's not empty; it's a map of targets to flags needed to build those targets; see https://github.com/hedronvision/bazel-compile-commands-extractor/blob/main/refresh_compile_commands.bzl